### PR TITLE
Rb vector translation

### PR DESCRIPTION
**Pull request by stanleykywu**

### CODE REVIEW

File: .eslintcache
Comment (krisharma) [inline .eslintcache:1]: Remember that we need to remove this file entirely from the repo, to do this:
`rm .eslintcache` in the terminal and then commit the deletion of the whole file rather than just the line!

File: package-lock.json
Comment (krisharma) [inline package-lock.json:13]: We shouldn't have a `package-lock.json` file since we are using yarn as our package manager - this means that you should do `rm package-lock.json` commit the deletion of the file, and then run `yarn install` to ensure that `yarn.lock` is up to date!

**Review by krisharma — CHANGES_REQUESTED** [review]
Most things look great! Few small changes that should be quick, rerequest my review once fixed!

**Comment by krisharma** [comment]
@maxpinheiro opened a new ticket related to this branch so I want to wait until we merge this, we'll likely want to merge off of this branch and make the requested changes from Prof Mingolla before we merge - happy to talk about this if you think it's better to merge this though!

File: src/modules/perceptrons/rblattVectors/RblattVectorsDemo.tsx
Comment (krisharma) [inline src/modules/perceptrons/rblattVectors/RblattVectorsDemo.tsx:154]: This is a pretty large block of code - let's move it into a function

**Review by krisharma — CHANGES_REQUESTED** [review]

File: src/modules/perceptrons/rblattVectors/RblattVectorsDemo.tsx
Comment (maxpinheiro) [reply src/modules/perceptrons/rblattVectors/RblattVectorsDemo.tsx:6]: yeah it can be a constant, it was just set up this way when we started and didn't realize we should change it - on it 👍

File: package.json
Comment (jakeisnt) [inline package.json:9]: Make sure to pin any dependencies you add to the exact version by deleting the prefixed carrot or using `yarn add --exact`. Pulling in the changes from merge conflicts should fix most of this.

**Review by krisharma — APPROVED** [review]
Looks great, thanks! Merge away

**Pull request merged**
